### PR TITLE
bat: add German translation

### PR DESCRIPTION
**Pull request by danwolb02**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

This adds a German translation for the `bat` page.

### CODE REVIEW

**Review by navarroaxel — APPROVED** [review]

File: pages.de/common/bat.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+> Ausgabe und Verkettung von einzelnen Dateien.
+> Ein `cat`-Ersatz mit Syntax-Hervorhebung und Git-Integration.
+
+- Gebe den Inhalt einer Datei in der Standardausgabe aus:
+
+`bat {{file}}`
+
+- Verkette mehrere Dateien in eine Zieldatei:
+
+`bat {{file1}} {{file2}} > {{target_file}}`
```
Comment (schneiderl) [inline pages.de/common/bat.md:12]: I believe we should also be translating the command params.

Comment (schneiderl) [reply pages.de/common/bat.md:12]: That applied to all examples we have here.

**Review by schneiderl — CHANGES_REQUESTED** [review]
I've just left a minor comment for you. I do not know the language but sintax wise it looks good to me.

**Comment by danwolb02** [comment]
You're right, I'll change it.

**Review by schneiderl — APPROVED** [review]

**Pull request merged**